MN/Algoritmi/QR/Gram-Schmidt/GramSchimidt.m

```diff
@@ -0,0 +1,13 @@
+function mat = GramSchimidt(mat)
+  k = size(mat, 2);
+  assert(k >= 2, 'The input matrix must include more than one vector.');
+  for i = 1 : 1 : k
+    mat(:, i) = mat(:, i) / norm(mat(:, i));
+    for j = i + 1 : 1 : k
+      mat(:, j) = mat(:, j) - proj(mat(:, i), mat(:, j));
+    endfor
+  endfor
+  function w = proj(a, b)
```

Comment:
```matlab
function [w] = Proj(a, b)
```
also, muta-l in Algoritmi-ad-hoc si scrie la inceputul sursei acesteia "[USES] Algoritmi-ad-hoc/Proj.